assets/src/design-system/components/input/index.js

```diff
@@ -157,29 +152,14 @@ export const Input = forwardRef(
     const inputRef = useRef(null);
     const [focused, setFocused] = useState(false);
```

Comment:
It could but since we're using `focused` in here then it would also need to return `focused` and it didn't feel like it would make sense to get that value from event handlers. Perhaps it's OK though with renaming to `isFocused`.